app/views/step_by_step_pages/show.html.erb

```diff
@@ -20,57 +20,37 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render "govuk_publishing_components/components/summary_list", @step_by_step_page_presenter.summary_list_params %>
 
-    <table class="govuk-table step-list">
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th class="govuk-table__header" scope="col">Step</th>
-          <th class="govuk-table__header" scope="col">Title</th>
-          <% if @step_by_step_page.can_be_edited? %>
-            <th class="govuk-table__header govuk-table__header--numeric" scope="col">Actions</th>
-          <% end %>
-        </tr>
-      </thead>
-
-      <tbody class="govuk-table__body">
-        <%
-          step_number = 1
-        %>
-        <% if @step_by_step_page.steps.present? %>
-          <% @step_by_step_page.steps.each do |step| %>
-            <tr class="govuk-table__row">
-              <th class="govuk-table__header" scope="row">
-                <% if step.logic == 'number' %>
-                  <%= step_number %>
-                  <% step_number += 1 %>
-                <% else %>
-                  <%= step.logic %>
-                <% end %>
-              </th>
-
-              <th class="govuk-table__header step-list__title" scope="row">
-                <%= step.title %>
-                <div class="step-broken-links__summary">
-                  <%= render 'shared/steps/broken-links-summary', step: step %>
-                </div>
-              </th>
-
-              <% if @step_by_step_page.can_be_edited? %>
-                <td class="govuk-table__cell govuk-table__cell--numeric">
-                  <%= link_to("Edit", edit_step_by_step_page_step_path(@step_by_step_page.id, step.id), class: "gem-c-button govuk-button gem-c-button--inline") %>
-                  <%= link_to("Delete", step_by_step_page_step_path(@step_by_step_page.id, step.id), method: 'delete', data: { confirm: "Are you sure?" }, class: "gem-c-button govuk-button govuk-button--warning gem-c-button--inline") %>
-                </td>
-              <% end %>
-            </tr>
-          <% end %>
-        <% else %>
-          <tr class="govuk-table__row">
-            <td class="govuk-table__cell" colspan="3">
-              <p class="govuk-hint">No steps have been added yet.</p>
-            </tr>
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
+    <%
+      step_number = 0
+      steps = @step_by_step_page.steps.map do |step|
+        step_number = step_number + 1 if step.logic == 'number'
+        {
+          field: step.logic == 'number' ? step_number : step.logic,
+          value: "#{step.title} #{render 'shared/steps/broken-links-summary', step: step}".html_safe,
```

Comment:
What is `value` supposed to be? Why is "render" being called here?

Reply:
It feels like we're trying to force summary-list component to act in a way it's not designed to. Is there scope for updating summary list to accept an html block, similar to inset-text?
cc @alex-ju

Reply:
After talking IRL, we've decided to:

1. Move `step.title` to the `broken-links-summary` partial,
2. Stop rendering `broken-links-summary` on the Step page as the text is very small and doesn't seem to be adding much value. If there are broken links, the broken link report is rendered, so the users are aware if something needs fixing.

This will allow `value` to be set to: `render('shared/steps/broken-links-summary', step: step)`

Reply:
I think `_broken-links-summary` should probably be renamed to `_summary` now as that's a more accurate description of what it is.